components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/impl/ApisApiServiceImpl.java

```diff
@@ -64,7 +66,7 @@
      */
     @Override
     public Response apisApiIdCommentsCommentIdDelete(String commentId, String apiId, String ifMatch,
-            String ifUnmodifiedSince, Request request) throws NotFoundException {
+                                                     String ifUnmodifiedSince, Request request) throws NotFoundException {
```

Comment:
Formatting issues. I think this line has gone beyond 120 mark. Check other places as well.